pages/common/git-rebase.md

```diff
@@ -16,6 +16,10 @@
 
 `git rebase --continue`
 
+- Skip a patch and restart the rebase that was interrupted by a merge failure:
```

Comment:
This implies that it's we're specifying a specific rebase to restart, as if to say that you can do multiple rebases at the same time and tell it which one to continue. Perhaps this would read better?

```suggestion
- Skip a patch and restart a rebase that was interrupted by a merge failure:
```

Maybe I'm being too picky.

Reply:
I agree it's reads more naturally with "a rebase" instead of "the rebase".

Reply:
Also, I'm not sure "skip a patch" is clear enough language, and the "restart" may be misleading. How about something along the following lines?

> Continue a rebase that was paused due to a merge conflict, by skipping the problematic step

or

> Continue a rebase that has stopped due to merge conflicts, by ignoring the conflicted commit

Of course, these are just suggestions. Feel free to play around with the word and phrasing choices above, or only pick some of their elements into your original sentence, if they don't immediately click for you.

Reply:
Ah, good point! I couldn't put my finger on what else it was. I like the 1st one best there.